src/website/src/pages/ec/components/navigation/inpage-navigation/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Inpage navigation
+defaultTab: usage
+status: ready
+playground:
+  system: ec
+  path: /story/components-navigation-in-page-navigation--default
```

Comment:
```suggestion
  path: /story/components-navigation-inpage-navigation--default
```

Reply:
This one is also used when clicking on the "Playground" tab, which leads to the storybook story.